commands/blacklist.js

```diff
@@ -27,6 +27,14 @@ exports.run = async (client, message, args) => {
     await client.db.updateUser(user.data)
 
     message.channel.send(`${greentick} ${client.utils.escapeMarkdown(user.tag)} (\`${user.id}\`) has been blacklisted`)
+    
+    //leave his server
+    try {
```

Comment:
Bet you know already it errored :kappalul:

Reply:
ew?